pallets/utxo/src/lib.rs

```diff
@@ -82,6 +117,7 @@ pub mod pallet {
 
     pub trait WeightInfo {
         fn spend(u: u32) -> Weight;
+        fn tokens_create(u: u32) -> Weight;
```

Comment:
This is minor thing but shouldn't this be `token_create`?

Reply:
Good question. @muursh is there we create token or tokenS? For example, we create 1000 tokens with a new token ID. Help us with proper English please =)

Reply:
Okay I say the singular token makes more sense here.

I think token_create is the better wording

Reply:
Done